be/src/olap/tablet.cpp

```diff
@@ -2719,6 +2719,8 @@ Status Tablet::calc_delete_bitmap(RowsetSharedPtr rowset,
                         ++row_id;
                         continue;
                     }
+                    // some unexpected error
+                    return st;
```

Comment:
We should check whether `st` is an error that was `NotFound`. If it not NotFound, then we can return.

Reply:
`_check_pk_in_pre_segments` won't return a `NotFound` status 😏